packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -196,7 +328,7 @@ static void InterpretInfoWindow(id<FLTGoogleMapMarkerOptionsSink> sink, NSDictio
     if (iconData.count == 3) {
       image = [UIImage imageNamed:[registrar lookupKeyForAsset:iconData[1]]];
       NSNumber *scaleParam = iconData[2];
```

Comment:
This is being assigned to an NSNumber, but the type is checked in the scale method which makes me think the type isn't actually known here (is this method channel data that may be `NSNull`?). I would leave a TODO here, as type abuse is dangerous.